### ISSUE TITLE
Mypy crashes on invalid superclass list

### ISSUE THREAD

**Issue by sandervoerman** [python/mypy#1794]
Type checking the following invalid Python code causes mypy to crash:

```python
class A:
    pass

class B(A):
    pass

class C(B):
    pass

class D(A, B):  # This raises a TypeError
    pass

class E(C, D):
    pass
```

Instead of recognizing and reporting the invalid superclass list of `D`, mypy crashes with the following traceback: 

```
Traceback (most recent call last):
  File "C:\Program Files\Python35\Scripts\mypy", line 6, in <module>
    main(__file__)
  File "C:\Program Files\Python35\lib\site-packages\mypy\main.py", line 54, in main
    res = type_check_only(sources, bin_dir, options)
  File "C:\Program Files\Python35\lib\site-packages\mypy\main.py", line 102, in type_check_only
    python_path=options.python_path)
  File "C:\Program Files\Python35\lib\site-packages\mypy\build.py", line 211, in build
    dispatch(sources, manager)
  File "C:\Program Files\Python35\lib\site-packages\mypy\build.py", line 1336, in dispatch
    process_graph(graph, manager)
  File "C:\Program Files\Python35\lib\site-packages\mypy\build.py", line 1467, in process_graph
    process_stale_scc(graph, scc)
  File "C:\Program Files\Python35\lib\site-packages\mypy\build.py", line 1497, in process_stale_scc
    graph[id].type_check()
  File "C:\Program Files\Python35\lib\site-packages\mypy\build.py", line 1316, in type_check
    manager.type_checker.visit_file(self.tree, self.xpath)
  File "C:\Program Files\Python35\lib\site-packages\mypy\checker.py", line 426, in visit_file
    self.accept(d)
  File "C:\Program Files\Python35\lib\site-packages\mypy\checker.py", line 476, in accept
    typ = node.accept(self)
  File "C:\Program Files\Python35\lib\site-packages\mypy\nodes.py", line 650, in accept
    return visitor.visit_class_def(self)
  File "C:\Program Files\Python35\lib\site-packages\mypy\checker.py", line 1068, in visit_class_def
    self.check_multiple_inheritance(typ)
  File "C:\Program Files\Python35\lib\site-packages\mypy\checker.py", line 1093, in check_multiple_inheritance
    if not (base1 in base2.mro or base2 in base1.mro):
AttributeError: 'NoneType' object has no attribute 'mro'

*** INTERNAL ERROR ***

mypybug.py:20: error: Internal error -- please report a bug at https://github.com/python/mypy/issues

NOTE: you can use "mypy --pdb ..." to drop into the debugger when this happens.
```

I am running mypy 0.4.2 and Python 3.5.2.
